tools/template-tool/Azure.Sdk.Tools.Template/Azure.Sdk.Tools.Template.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is a template project that helpful in the tools repo? I feel like we could pick any of our tools and do a test release if needed.

Reply:
I have removed that.